### PR TITLE
Added gunivers.net network and kanishiel.com

### PR DESCRIPTION
**Pull request by fantomitechno**
I love PreMiD so I tried to implemented the organisations' web sites, I'm in.

### CODE REVIEW

File: websites/G/Gunivers/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "project.gunivers.net"
+    ],
+    "description": {
+      "en": "Gunivers est une communauté autour de minecraft d'origine française, mais à destination d'un public français ou international selon les projets.",
+      "fr": "Gunivers is a minecraft community of French origin, but intended for a french or international audience depending of the project."
+    },
+    "service": "Gunivers",
+    "version": "1.0.0",
+    "logo": "https://cdn.discordapp.com/emojis/767693611516821524.png?v=1",
+    "thumbnail": "https://cdn.discordapp.com/emojis/801804688302276628.png?v=1",
```
Comment (Bas950) [inline websites/G/Gunivers/dist/metadata.json:19]: upload to imgur

File: websites/G/Gunivers/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "fantomitechno 🦊#8888",
+      "id": "563749920683720709"
```
Comment (Bas950) [inline websites/G/Gunivers/dist/metadata.json:5]: remove your tag (#8888)

File: websites/G/Gunivers/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
```
Comment (Bas950) [inline websites/G/Gunivers/dist/metadata.json:2]: Use the latest schema

**Review by Bas950 — CHANGES_REQUESTED** [review]
Same issues in the other presence

**Comment by fantomitechno** [comment]
I will modify this as faster I can

**Comment by fantomitechno** [comment]
Fixed

File: websites/G/Gunivers/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      "status.gunivers.net",
+      "project.gunivers.net"
+    ],
+    "description": {
+      "en": "Gunivers est une communauté autour de minecraft d'origine française, mais à destination d'un public français ou international selon les projets.",
+      "fr": "Gunivers is a minecraft community of French origin, but intended for a french or international audience depending of the project."
+    },
+    "service": "Gunivers",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/pjuZo1A.png",
```
Comment (ririxidev) [inline websites/G/Gunivers/dist/metadata.json:18]: logo must be 512x512 or more

File: websites/G/Gunivers/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    "thumbnail": "https://i.imgur.com/sNN579z.png",
```
Comment (ririxidev) [inline websites/G/Gunivers/dist/metadata.json:19]: it should be promotional banner or screenshot of website

**Review by ririxidev — CHANGES_REQUESTED** [review]
yikes

**Comment by fantomitechno** [comment]
I will update this in 2 hors (I cant before)

**Comment by fantomitechno** [comment]
Modified

**Comment by fantomitechno** [comment]
I find some way to improve the gunivers.net presence, I used the link to have the articles or chronicles names but I learned a way to read the page

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by fantomitechno** [comment]
Gunivers : 
image

image

image

KaniShiel : 

image

image

image

File: websites/K/KaniShiel/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      "id": "563749920683720709"
+    },
+    "url": "kanishiel.com",
+    "description": {
+      "en": "KaniShiel is a discord bot that can host multiple games on an indefinite number of simultaneous servers.",
+      "fr": "KaniShiel est un bot de jeu qui peut animer de multiple jeu sur un nombre indéfini de serveur simultané."
+    },
+    "service": "KaniShiel",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/e9IeASG.png",
```
Comment (Bas950) [inline websites/K/KaniShiel/dist/metadata.json:14]: logo must at least be 512x512

**Review by Bas950 — COMMENTED** [review]
Upload proof of it working

**Comment by fantomitechno** [comment]
Done

You need more proof of it working or these aren't good ?

**Comment by Bas950** [comment]
> Done
> 
> You need more proof of it working or these aren't good ?

No sorry forgot to delete the old text

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**